utils/post.js

```diff
@@ -326,12 +326,13 @@ const deleteExpiredPostFromFeed = (item, feed_id) => {
   }
 };
 
-const modifyNewItemAnonymity = (newItem) => {
+const modifyNewItemAnonymity = (newItem, isBlurredPost = false) => {
   if (newItem.anonimity) {
     newItem.actor = {};
     newItem.to = [];
     newItem.origin = null;
     newItem.object = '';
+    newItem.isBlurredPost = isBlurredPost;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `modifyNewItemAnonymity` function has been updated to include the `isBlurredPost` parameter. However, it seems like the `isBlurredPost` property is being added to `newItem` regardless of whether `newItem.anonimity` is true or not. If the `isBlurredPost` property is only relevant for anonymous posts, it should be inside the if block.
```diff
-  newItem.isBlurredPost = isBlurredPost;
   }
+  newItem.isBlurredPost = isBlurredPost;
   return newItem;
 };
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->